src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Symbolic/SymbolicRegexBuilder.cs

```diff
@@ -488,5 +501,155 @@ private DfaMatchingState<TElement> MakeNewState(DfaMatchingState<TElement> state
                 return state;
             }
         }
+
+        /// <summary>
+        /// Initialize or reinitialize the NFA state map and the NFA transition table for Antimirov mode.
+        /// </summary>
+        internal void InitializeAntimirovMode()
+        {
+            if (_antimirov)
+            {
+                _antimirovStateArray = Array.Empty<int>();
+                _antimirovDelta = Array.Empty<List<int>>();
+                _antimirovStateArrayInverse.Clear();
+            }
+            else
+            {
+                _antimirov = true;
+            }
+        }
```

Comment:
This method is only called from one place, where _antimirov is false.  Can we delete this and just change the call site to set _antimirov = true?